MyCryptoHQ/MyCrypto
# Create script to update tokens list from ethereum-lists/tokens

Triage temporary solution for #969.

### Description

Right now we've just been keeping up to date with MyCryptoHQ/mycrypto.com for token lists changes, but the future plan is to use [ethereum-lists/tokens](https://github.com/ethereum-lists/tokens) as our source of tokens. The problem is its format is kind of spread out and not easy to import directly to our repo. This PR adds a script that fetches its output from IPFS, imports and formats the tokens to match our format, and stages the tokens for commit. This won't stop us from manually updating, but should make the process much easier.

One concern of this change is that our token list has ballooned from ~90 to 513 with this. This'll be a much larger load on our node, and probably includes a lot of junk tokens. Not to mention, some of them don't use proper symbol acronyms, and instead have rather long names. We may want to either audit `ethereum-lists/tokens` and clean out any useless ones, or implement a blacklist of tokens to exclude.

~Also of note, I'm not sure decimals or amounts are being handled properly right now with tokens. See screenshot below.~ Fixed, see comment below.

### Changes

* Add a script to a new `scripts/` folder that fetches tokens.
* Updated tokens list with what was fetched from the aforementioned script.

### Steps to Test

1. Checkout `eth-lists-script`
2. Run `npm run update:tokens`
3. Confirm the script runs without fail
4. Confirm no changes are made, since the token lists should be the same

### Screenshots

<img width="493" alt="screen shot 2018-03-04 at 5 01 05 pm" src="https://user-images.githubusercontent.com/649992/36951162-54b25dd2-1fce-11e8-807d-6726f88f480e.png">

<img width="428" alt="screen shot 2018-03-04 at 5 07 15 pm" src="https://user-images.githubusercontent.com/649992/36951187-a3800360-1fce-11e8-9fb3-88d4b904a8b0.png">

^ Are these amounts correct? Is something wrong with decimals?
